modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/jwt/JWTRevocationTestCase.java

```diff
@@ -0,0 +1,295 @@
+        //Call the revoke Endpoint
+        URL revokeEndpointURL = new URL(gatewayUrlsWrk.getWebAppURLNhttp() + "revoke");
+        org.wso2.carbon.automation.test.utils.http.client.HttpResponse httpResponse;
+        try {
+            httpResponse = HTTPSClientUtils.doPost(revokeEndpointURL, input, authenticationRequestHeaders);
+            assertEquals(httpResponse.getResponseCode(), 200);
+        } catch (Exception e) {
+            Assert.fail("Should not throw any exceptions" + e);
+        }
+        log.info("revokeRequestTestCase  Finished");
+    }
+
+    @Test(groups = { "wso2.am" }, description = "JWT revocation test", dependsOnMethods = "revokeRequestTestCase")
+    public void checkETCDForRevokedJTITestCase() throws Exception {
+        log.info("checkETCDForRevokedJTITTestCase  Initiated");
+        String etcdEndpointURLs = "https://localhost:9943/etcdmock/v2/keys/jti/2f3c1e3a-fe4c-4cd4-b049-156e3c63fc5d";
+        URL etcdEndpointURL = new URL("https://localhost:9943/etcdmock/v2/keys/jti/2f3c1e3a-fe4c-4cd4-b049-156e3c63fc5d");
+        Map<String, String> authenticationRequestHeaders = new HashMap<>();
+        authenticationRequestHeaders.put("Content-Type", "application/x-www-form-urlencoded");
+        String retrievedETCDJTIKey = "";
+        String input ="";
+        try {
+            HTTPSClientUtils.doPost(etcdEndpointURL, input, authenticationRequestHeaders);
+            JSONObject etcdResponse = new JSONObject(
+                    HTTPSClientUtils.doGet(etcdEndpointURLs, authenticationRequestHeaders).getData());
+            Iterator<String> keys = etcdResponse.keys();
+            while (keys.hasNext()) {
+                String key = keys.next();
+                if (key.equalsIgnoreCase("node")) {
+                    // This is to retrieve the nested json object
+                    JSONObject childObject = ((JSONObject) etcdResponse.get("node"));
+                    Iterator<String> childKeys = childObject.keys();
+                    while (childKeys.hasNext()) {
+                        String childKey = childKeys.next();
+                        // Iterating through the  child json object
+                        if (childKey.equalsIgnoreCase("key")) {
+                            retrievedETCDJTIKey = childObject.getString(childKey).substring(5);
+                        }
+                    }
+                }
+            }
+        } catch (JSONException e) {
+            Assert.fail("Should not throw any exceptions" + e);
+        }
+
+        jti= "2f3c1e3a-fe4c-4cd4-b049-156e3c63fc5d";
+        assertEquals(retrievedETCDJTIKey, jti);
+        log.info("checkETCDForRevokedJTITTestCase  Finished");
+    }
+
+    @Test(groups = { "wso2.am" }, description = "JWT revocation test", dependsOnMethods = "revokeRequestTestCase")
+    public void checkJMSTopicForRevokedJTITestCase() throws Exception {
+        log.info("checkJMSTopicForRevokedJTITTestCase  Initiated");
+
```

Comment:
Lets remove the extra lines in all the classes

Reply:
resolved